static/src/javascripts/projects/commercial/modules/article-body-adverts.ts

```diff
@@ -55,6 +55,16 @@ const insertAdAtPara = (
 		});
 };
 
+const isInRichLinksVariant = isInVariantSynchronous(
+	spacefinderOkr3RichLinks,
+	'variant',
+);
+
+const enableListItemsFix = isInVariantSynchronous(
+	spacefinderOkrMegaTest,
+	'variant',
+);
+
```

Comment:
I appreciate these conditionals are getting a bit hairy but for a temporary test setup it feels ok to me